src/awkward/_connect/cuda/cuda_kernels/awkward_IndexedArray_reduce_next_nonlocal_nextshifts_64.cu

```diff
@@ -6,23 +6,22 @@
 //     scan_in_array_k = cupy.empty(length, dtype=cupy.int64)
 //     scan_in_array_nullsum = cupy.empty(length, dtype=cupy.int64)
 //     cuda_kernel_templates.get_function(fetch_specialization(["awkward_IndexedArray_reduce_next_nonlocal_nextshifts_64_a", nextshifts.dtype, index.dtype]))(grid, block, (nextshifts, index, length, scan_in_array_k, scan_in_array_nullsum, invocation_index, err_code))
-//     scan_in_array_k = inclusive_scan(grid, block, (scan_in_array_k, invocation_index, err_code))
-//     scan_in_array_nullsum = inclusive_scan(grid, block, (scan_in_array_nullsum, invocation_index, err_code))
+//     scan_in_array_k = cupy.cumsum(scan_in_array_k)
+//     scan_in_array_nullsum = cupy.cumsum(scan_in_array_nullsum)
 //     cuda_kernel_templates.get_function(fetch_specialization(["awkward_IndexedArray_reduce_next_nonlocal_nextshifts_64_b", nextshifts.dtype, index.dtype]))(grid, block, (nextshifts, index, length, scan_in_array_k, scan_in_array_nullsum, invocation_index, err_code))
 // out["awkward_IndexedArray_reduce_next_nonlocal_nextshifts_64_a", {dtype_specializations}] = None
 // out["awkward_IndexedArray_reduce_next_nonlocal_nextshifts_64_b", {dtype_specializations}] = None
 // END PYTHON
 
 template <typename T, typename C>
 __global__ void
-awkward_IndexedArray_reduce_next_nonlocal_nextshifts_64_a(
-    T* nextshifts,
-    const C* index,
-    int64_t length,
-    int64_t* scan_in_array_k,
-    int64_t* scan_in_array_nullsum,
-    uint64_t invocation_code,
-    uint64_t* err_code) {
+awkward_IndexedArray_reduce_next_nonlocal_nextshifts_64_a(T* nextshifts,
+                                                          const C* index,
+                                                          int64_t length,
+                                                          int64_t* scan_in_array_k,
+                                                          int64_t* scan_in_array_nullsum,
+                                                          uint64_t invocation_code,
+                                                          uint64_t* err_code) {
```

Comment:
This originally had black/ruff-style formatting so that changes in diff would be more localized. For instance, a change in the name would only affect the line where the name is, but with this indenting, a change in the length of the name would change where all arguments are indented to.

This is not a major issue. It's very unlikely that we'll be changing names or argument lists, since it's all constrained by a specification and users don't see it. I'm just saying that there was a reason for the original formatting.

Reply:
Oh, since all the other kernels had the other formatting except these few so I changed these too. But I can fix that in another PR.